matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXMegolmEncryption.kt

```diff
@@ -275,7 +285,7 @@ internal class MXMegolmEncryption(
                 gossipingEventBuffer.add(
                         Event(
                                 type = EventType.ROOM_KEY,
-                                senderId = this.userId,
+                                senderId = this.myUserId,
```

Comment:
I think `this.` is not necessary anymore here

Reply:
done